### PR TITLE
Allow create access for SelfSubjectAccessReview

### PR DESCRIPTION
**Pull request by rottencandy**
Add create permission for `SelfSubjectAccessReview` to allow for RBAC checks in UI.

Test repo PR: https://github.com/codeready-toolchain/toolchain-e2e/pull/681

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

File: deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml [hunk 1]
```diff
@@ -158,6 +158,12 @@ objects:
     - watch
     - update
     - patch
+  - apiGroups:
+    - authorization.k8s.io
+    resources:
+    - selfsubjectaccessreview
```
Comment (MatousJobanek) [inline deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml:164]: correct me if I'm wrong, but it should be in the plural form, right?:
```suggestion
    - selfsubjectaccessreviews
```

Comment (MatousJobanek) [reply deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml:164]: btw, is `selfsubjectaccessreview`  the only one that is used by UI, or does it also use SubjectAccessReview?

Comment (alexeykazakov) [reply deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml:164]: Yes, it should be in the plural form indeed! I keep missing it when reviewing those PRs :(

Comment (rottencandy) [reply deploy/templates/nstemplatetiers/appstudio/spacerole_admin.yaml]: Updated!

The UI doesn't need to check access for users other than the current user, so SubjectAccessReview is not needed.

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**